project_subtask_default_display_project/views/project_task_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Ángel García de la Chica Herrera <angel.garcia@sygel.es>
+ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="project_subtask_default_project_view_task_form2" model="ir.ui.view">
+        <field name="name">project.subtask.default.project.view.task.form2</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='child_ids']" position='attributes'>
+                <attribute
+                    name="context"
+                >{'default_project_id': project_id, 'default_display_project_id': project_id, 'default_user_ids': user_ids, 'default_parent_id': id, 'default_partner_id': partner_id}</attribute>
```

Comment:
do you have to define the whole context? can't you add the key with the relevant value directly?